backend/vehicles/src/modules/dms/coms.service.ts

```diff
@@ -9,10 +9,10 @@ import { ReadFileDto } from './dto/response/read-file.dto';
 export class ComsService {
   constructor(private readonly httpService: HttpService) {}
 
-  async createObject(file: Express.Multer.File): Promise<ReadCOMSDto[]> {
-    const { buffer, originalname } = file;
+  async createObject(file: ArrayBuffer): Promise<ReadCOMSDto[]> {
```

Comment:
@praju-aot please let me know if you agree with this change. It seems quite a bit easier for us to deal with Array Buffers instead of passing Files

Reply:
created temp method using ArrayBuffer